code/modules/client/verbs/ooc.dm

```diff
@@ -317,8 +317,8 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 
 	var/message = "<span class='big'>You can add emphasis to your text by surrounding words or sentences in certain characters.</span>\n \
 		**bold**, and _italics_ are supported.\n\n \
-		<span class='big'>You can made custom saymods by doing <i>say 'screams- HELP IM DYING!'</i>. This works over the radio, and can be used to emote over the radio.</span>\n \
-		Example: say ';laughs maniacally!-' >> \[Common] Joe Schmoe laughs maniacally!"
+		<span class='big'>You can made custom saymods by doing <i>say 'screams/ HELP IM DYING!'</i>. This works over the radio, and can be used to emote over the radio.</span>\n \
+		Example: say ';laughs maniacally!/' >> \[Common] Joe Schmoe laughs maniacally!"
```

Comment:
Perhaps make it backslash? Slash is sometimes used.

Reply:
From what I know, backslashes are treated by the code as a method of saying "insert a new line here"

Reply:
No, that's `\n`. You can escape a `\` by putting `\\`. I think `\` is a fine replacement.

Reply:
I think a single backslash is reasonable and wouldn't collide with anything else.
You don't need to escape any characters in say.

Reply:
Right made it happen, however it now requires two backslashes instead to allow for a custom say.
So to make a "Joe Danger hums 'What a lovely shift'" you need two backslashes, only sending one backslash would break the message resulting in it ending up as "Joe Danger says 'humsWhat a lovely shift'"

Strangely enough it only takes one backslash to actually send a custom emote so "Joe Danger hums" is perfectly fine but the second you attach a message then you'll end up needing two of them.

Reply:
lmao what

Reply:
> lmao what

https://user-images.githubusercontent.com/59128051/215491711-6ac70879-8898-484a-a540-2eabd3dea131.mp4

Reply:
> You're using the unsupported verb input, my friend 😭

The WHAT